django/db/models/sql/where.py

```diff
@@ -129,12 +129,14 @@ def as_sql(self, compiler, connection):
             # Convert if the database doesn't support XOR:
             #   a XOR b XOR c XOR ...
             # to:
-            #   (a OR b OR c OR ...) AND (a + b + c + ...) == 1
+            #   (a OR b OR c OR ...) AND MOD(a + b + c + ..., 2) == 1
```

Comment:
Could you please include a small text clarification explaining the change? For example:

`The result of an n-ary xor is true if and only if an odd number of arguments are true.`